lib/Alchemy/Phrasea/Core/Provider/ORMServiceProvider.php

```diff
@@ -12,570 +12,207 @@
-                        break;
-                }
-            }
-            $config->setMetadataDriverImpl($chain);
+        $reader = new AnnotationReader();
+        $reader = new CachedReader($reader, $cache);
 
-            foreach ((array) $options['types'] as $typeName => $typeClass) {
-                if (Type::hasType($typeName)) {
-                    Type::overrideType($typeName, $typeClass);
-                } else {
-                    Type::addType($typeName, $typeClass);
-                }
-            }
-
-            return $config;
-        });
+        $driver = new AnnotationDriver($reader, [
+            $app['root.path'] . '/vendor/gedmo/doctrine-extensions/lib/Gedmo/Translatable/Entity/MappedSuperclass',
+            $app['root.path'] . '/vendor/gedmo/doctrine-extensions/lib/Gedmo/Loggable/Entity/MappedSuperclass',
+            $app['root.path'] . '/vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity/MappedSuperclass',
+            $app['root.path'] . '/lib/Alchemy/Phrasea/Model/Entities'
+        ]);
 
-        $app['orm.ems.options'] = $app->share(function() use ($app) {
-            if (false === $app['phraseanet.configuration']->isSetup()) {
-                return [];
-            }
+        return $driver;
+    }
 
-            return array_merge(
-                $app['orm.em.appbox.options'],
-                $app['orm.ems.databox.options'],
-                $app['orm.em.fixture.options'],
-                $app['orm.em.test.options']
+    private function getCacheDriver(PhraseaApplication $app)
+    {
+        $conf = $app['conf']->get(['main', 'cache']);
 
-            );
-        });
+        return isset($conf['type']) ? $conf['type'] : 'ArrayCache';
+    }
 
-        /**
-         * Check database connection information
-         */
-        $app['db.info'] = $app->protect(function(array $info){
-            if (!isset($info['driver'])) {
-                $info['driver'] = 'pdo_mysql';
-            }
+    private function getCacheOptions(PhraseaApplication $app)
+    {
+        $conf = $app['conf']->get(['main', 'cache']);
 
-            if (!isset($info['charset'])) {
-                $info['charset'] = 'utf8';
-            }
+        return isset($conf['options']) ? $conf['options'] : [];
+    }
 
-            switch ($info['driver']) {
-                case 'pdo_mysql':
-                    foreach (array('user', 'password', 'host', 'dbname', 'port') as $param) {
-                        if (!array_key_exists($param, $info)) {
-                            throw new InvalidArgumentException(sprintf('Missing "%s" argument for database connection using driver %s', $param, $info['driver']));
-                        }
+    private function validateConnectionSettings(array $parameters)
+    {
+        if (!isset($parameters['driver'])) {
+            $parameters['driver'] = 'pdo_mysql';
+        }
+
+        if (!isset($parameters['charset'])) {
+            $parameters['charset'] = 'utf8';
```

Comment:
SQLite does not know about this charset encoding.
The Recommended for MySQL is utf8mb4.